enterprise/frontend/src/embedding-sdk/test/auth-flow.unit.spec.tsx

```diff
@@ -26,7 +27,7 @@ const AUTH_PROVIDER_URL = "http://auth-provider:3000/sso/metabase";
 const MOCK_API_KEY = "mock-api-key";
 const USER_CURRENT_URL = `${METABASE_INSTANCE_URL}/api/user/current`;
 const MOCK_SESSION = {
-  exp: 1729761473,
+  exp: Number.MAX_SAFE_INTEGER,
```

Comment:
Needed otherwise it made 3 requests to the auth endpoint as the session was already expired